ajax/libs/jquery.sumoselect/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.sumoselect",
+  "version": "3.0.2",
+  "description": "A jQuery plugin that progressively enhances an HTML Select Box into a Single/Multiple option dropdown list",
```

Comment:
@pvnr0082t I used `description` and `keywords` in `package.json`, and I revised them. Please help me review this, thanks!